runtime/bref/0.2/serverless.yaml

```diff
@@ -0,0 +1,50 @@
+# Read the documentation at https://www.serverless.com/framework/docs/providers/aws/guide/serverless.yml/
+service: my-symfony-app
+configValidationMode: error
+
+provider:
+    name: aws
+    # The AWS region in which to deploy (us-east-1 is the default)
+    region: us-east-1
+    # The stage of the application, e.g. dev, production, staging… ('dev' is the default)
+    stage: dev
+    runtime: provided.al2
+    environment:
+        # Symfony environment variables
+        APP_ENV: prod
+        APP_RUNTIME: Runtime\Bref\Runtime
+        BREF_LOOP_MAX: 1 # Increase to keep the Lambda process alive between requests
```

Comment:
What about adding (commented): `SYMFONY_DECRYPTION_SECRET: ${ssm:/stripe-notifications/prod-decrypt-private-key}`

Reply:
Great idea

Reply:
Mentionning `stripe` here seems a bit odd to me. I would go with something more generic like `${ssm:/your-app/prod-decrypt-private-key}`